main/main.c

```diff
@@ -554,6 +554,14 @@ void interactiveLoop (cookedArgs *args CTAGS_ATTR_UNUSED, void *user CTAGS_ATTR_
 			openTagFile ();
 			if (size == -1)
 			{					/* read from disk */
+#ifdef HAVE_SECCOMP
+				struct interactiveModeArgs *iargs = user;
+				if (iargs->secure) {
+					error (FATAL,
+						   "invalid request in secure submode: reading file contents from a file is limited");
+					goto next;
```

Comment:
I expect this error message will reduce the bug reports we will get.
People never read the documents we wrote.